src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -57,6 +57,11 @@ def load_arguments(self, _):
         type=int,
         help='Maximum number of items to return. Must be a positive integer. Default to 100.'
     )
+    retention_days_arg_type = CLIArgumentType(
+        options_list=['--retention-days'],
+        type=int,
+        help='Number of days to retain the soft delete enabled App Configuration after deleting. Must be a positive integer between 0 and 7.'
```

Comment:
Do we need to add verification on the client side for it's valid value?

Reply:
We have intentionally left out the validation for this value on the client side to prevent tight coupling with the server.

Reply:
OK~